### PR TITLE
Update tiup-cluster.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9893
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -21,36 +21,38 @@ Starting component `cluster`: /home/tidb/.tiup/components/cluster/v1.10.0/cluste
...
-      --wait-timeout int  Timeout of waiting the operation
-      --ssh-timeout int   Timeout in seconds to connect host via SSH, ignored for operations that don't need an SSH connection. (default 5)
-  -y, --yes               Skip all confirmations and assumes 'yes'
+  -c, --concurrency int     max number of parallel tasks allowed (default 5)
+      --format string       (EXPERIMENTAL) The format of output, available values are [default, json] (default "default")
+  -h, --help                help for tiup
+      --ssh string          (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.
+      --ssh-timeout uint    Timeout in seconds to connect host via SSH, ignored for operations that don't need an SSH connection. (default 5)
+  -v, --version             version for tiup
+      --wait-timeout uint   Timeout in seconds to wait for an operation to complete, ignored for operations that don't fit. (default 120)
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:54]: TThe second sentence is missing in the Chinese
```suggestion
      --wait-timeout uint   Timeout in seconds to wait for an operation to complete. Inapplicable operations are ignored. (defaults to `120`)
```

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -369,7 +371,7 @@ Flags:
       --transfer-timeout int   Timeout in seconds when transferring PD and TiKV store leaders (default 300)
 
 Global Flags:
-      --native-ssh        Use the system's native SSH client
+      --ssh string          (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:374]:
```suggestion
      --ssh string          (Experimental) The executor type. Optional values are 'builtin', 'system', and 'none'.
```

File: tiup/tiup-cluster.md [hunk 3]
```diff
@@ -465,7 +467,7 @@ Flags:
...
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:470]: ditto

File: tiup/tiup-cluster.md [hunk 4]
```diff
@@ -517,7 +519,7 @@ Flags:
   -r, --rename NAME        Rename the imported cluster to NAME
 
 Global Flags:
-      --native-ssh        Use the system's native SSH client
+      --ssh string        (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:522]: ditto

File: tiup/tiup-cluster.md [hunk 5]
```diff
@@ -21,36 +21,38 @@ Starting component `cluster`: /home/tidb/.tiup/components/cluster/v1.10.0/cluste
...
   help        Help about any command
 
 Flags:
-  -h, --help              help for cluster
-      --native-ssh        Use the system's native SSH client
-      --wait-timeout int  Timeout of waiting the operation
-      --ssh-timeout int   Timeout in seconds to connect host via SSH, ignored for operations that don't need an SSH connection. (default 5)
-  -y, --yes               Skip all confirmations and assumes 'yes'
+  -c, --concurrency int     max number of parallel tasks allowed (default 5)
+      --format string       (EXPERIMENTAL) The format of output, available values are [default, json] (default "default")
```
[the remark this answers is not in the document]
Comment (qqqdan) [reply tiup/tiup-cluster.md:49]: 回头我加一下中文

File: tiup/tiup-cluster.md (hunk 4 above)
Comment (qqqdan) [inline tiup/tiup-cluster.md:522]:
```suggestion
      --ssh string        (Experimental) The executor type. Optional values are 'builtin', 'system', and 'none'.
```

File: tiup/tiup-cluster.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (qqqdan) [reply tiup/tiup-cluster.md:48]: 中文那边已经加上了

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**